script.js

```diff
@@ -938,3 +938,621 @@ if ($(window).width() <= 700) {
 $(function(){
    $.ui.dialog.prototype._focusTabbable = function(){};
 });
+
```

Comment:
OK for loading everywhere; if we do not have choice for now. But +1 to use a specific file, will be easier to load when required in the future ;)